src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -1365,7 +1369,8 @@
       (into #{} (mapcat identity)
         [(completion-candidates-for-ns full-ns false)
          (completion-candidates-for-ns (add-macros-suffix full-ns) false)
-         (completion-candidates-for-js-sources full-ns)]))
+         (when-not (get-namespace full-ns)
```

Comment:
Let's move this check to the Closure candidates function please

Reply:
I can do it, but this expresses better that we don't want JS completions for cljs namespaces IMHO. Unless there is something I am missing of course

Reply:
just looks a bit clunky here. I'd prefer it localized

Reply:
Ok no probs, will move it